create-projects/Jenkinsfile

```diff
@@ -1,55 +1,62 @@
-def bitbucketHost = env.BITBUCKET_HOST;
-def dockerRegistry
-
-node {
-  dockerRegistry = env.DOCKER_REGISTRY
-}
-def conts = containerTemplate( 
-            name: 'jnlp',
-            image: "${dockerRegistry}/cd/jenkins-slave-base",
-            workingDir: '/tmp',
-            alwaysPullImage: true,
-            args: ''
-        )
-
-def podLabel = "create-projects-" + projectId;
-podTemplate(
-      label: podLabel,
-      cloud: 'openshift',
-      containers: [conts],
-      volumes: [],
-      serviceAccount: 'jenkins'
-    ) {
-        node(podLabel) {
-          stage('checkout'){
-            checkout scm
-          }
-          stage('checkout ods-config'){
-             checkout([
-                $class: 'GitSCM',    
-                branches: [[name: '*/production']],
-                doGenerateSubmoduleConfigurations: false, 
-                extensions: [[$class: 'RelativeTargetDirectory', relativeTargetDir: 'ods-config']],
-                submoduleCfg: [], 
-                userRemoteConfigs: [[credentialsId: 'prov-cd-cd-user-with-password', url: bitbucketHost + '/scm/opendevstack/ods-configuration.git']] 
-            ])
-          }
-
-
-
-        stage('creating openshift projects') {
-            sh "sh ./create-projects/create-projects.sh -p ${projectId} -n ${nexusHost} -a ${projectAdmins} -e ${projectGroups}";
-        }
-        stage('creating jenkins') {
-          sh "env";
-           withEnv(["pipelineTriggerSecret=${pipelineTriggerSecret}"]) {
-              sh "sh ./create-projects/create-cd-jenkins.sh -p ${projectId}";
-           }
-        }
-      }
-}
\ No newline at end of file
+// BuildConfig environment variables
+def projectId = env.PROJECT_ID
+def projectAdmins = env.PROJECT_ADMIN
+def projectGroups = env.PROJECT_GROUPS
+def pipelineTriggerSecret = env.PIPELINE_TRIGGER_SECRET
+
+// Jenkins DeploymentConfig environment variables
+def bitbucketHost
+def dockerRegistry
+node {
+  dockerRegistry = env.DOCKER_REGISTRY
+  bitbucketHost = env.BITBUCKET_HOST
+}
+
+def conts = containerTemplate(
+  name: 'jnlp',
+  image: "${dockerRegistry}/cd/jenkins-slave-base",
+  workingDir: '/tmp',
+  alwaysPullImage: true,
+  args: ''
+)
+def podLabel = "create-projects-${projectId}";
+
+podTemplate(
+  label: podLabel,
+  cloud: 'openshift',
+  containers: [conts],
+  volumes: [],
+  serviceAccount: 'jenkins'
+) {
+  node(podLabel) {
+
+    stage('checkout'){
+      checkout scm
+    }
+
+    stage('checkout ods-config'){
+      checkout([
+        $class: 'GitSCM',
+        branches: [[name: '*/master']],
+        doGenerateSubmoduleConfigurations: false,
+        extensions: [[
+          $class: 'RelativeTargetDirectory',
+          relativeTargetDir: 'ods-config'
+        ]],
+        submoduleCfg: [],
+        userRemoteConfigs: [[
+          credentialsId: 'prov-cd-cd-user-with-password',
+          url: "https://${bitbucketHost}/scm/opendevstack/ods-configuration.git"
```

Comment:
@kiwo This is the variable I was talking about ...

Reply:
Could you please confirm that your `bitbucketHost`(which simply is `BITBUCKET_HOST` of the `DeploymentConfig`) has no scheme? If that is the case, I assume that Jenkins defaults to `http://` if no scheme has been given.

Reply:
@kiwo Also, is your `BITBUCKET_HOST` simply `192.168.56.31:7990` as in the sample config?